userspace/libsinsp/sinsp.cpp

```diff
@@ -1967,31 +1968,6 @@ void sinsp::set_proc_scan_log_interval_ms(uint64_t val)
 	m_proc_scan_log_interval_ms = val;
 }
 
-void sinsp::set_sinsp_stats_v2_enabled()
-{
-	if (m_sinsp_stats_v2 == nullptr)
-	{
-		m_sinsp_stats_v2 = std::make_unique<sinsp_stats_v2>();
-		m_sinsp_stats_v2->m_n_noncached_fd_lookups = 0;
```

Comment:
I don't see that we init the values elsewhere now. We should keep that to avoid bogus numbers.

Reply:
Nope, becase we use `make_shared` that will 0 initialize everything (check eg: https://stackoverflow.com/questions/45366323/does-make-shared-do-a-default-initialization-zero-init-for-each-member-variabl)
Also, multiple tests would fail it that wasn't the case!

Reply:
Thanks!